.github/pull_request_template.md

```diff
@@ -0,0 +1,53 @@
+## Description
+
+<!--
+Describes the nature of your changes. If they are substantial, you should
+further subdivide this into a section describing the problem you are solving and
+another describing your solution.
+-->
+
+## Related issue(s)
+
+<!--
+- Short description of how the PR relates to the issue, including an issue link.
+For example
+- Fixed #100500 by adding lenses to exported items
+
+Write 'None' if there are no related issues (which is discouraged).
+-->
+
+Resolves #
+
+## :white_check_mark: Checklist for your Pull Request
+
+<!--
+Ideally a PR has all of the checkmarks set.
+
+If something in this list is irrelevant to your PR, you should still set this
+checkmark indicating that you are sure it is dealt with (be that by irrelevance).
+
+If you don't set a checkmark (e. g. don't add a test for new functionality),
+you must be able to justify that.
+-->
+
+#### Related changes (conditional)
+
+- Tests
+  - [ ] If I added new functionality, I added tests covering it.
+  - [ ] If I fixed a bug, I added a regression test to prevent the bug from
+        silently reappearing again.
+
+- Documentation
+  - [ ] I checked whether I should update the docs and did so if necessary:
+    - [README](../tree/master/README.md)
+    - Haddock
+
+- Public contracts
+  - [ ] I added an entry to the [changelog](../tree/master/CHANGES.md) if my changes are visible to the users
```

Comment:
No changelog at this stage I think.

Reply:
I believe that all `Public contracts` section can be removed at this stage